styles/blocks/section-2.json

```diff
@@ -0,0 +1,37 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/theme.json",
+	"version": 3,
+	"slug": "section-2",
```

Comment:
Food for thought, `is-style-section-2` will be the class name here. Just thinking, should they be `is-stye-pattern-2`—as these section styles are generally applied at the "pattern" level?

Reply:
My two cents: Although they're going to be generally applied at the pattern level, it _could not_ be 100% of the cases, and for those edge cases, it can be confusing to have something named pattern when it's not.

At a theme level like this, it will make sense. But a site admin can create a group of blocks and apply styles without the whole thing being a pattern. I believe this section is more generic.

Reply:
This goes back to this discussion: https://github.com/WordPress/twentytwentyfive/issues/3

I would avoid tying the slug to patterns just because that's where they're generally being used in this theme. Section/block styles can be used at any level (on the pattern wrapper, in sub-sections of patterns, in templates, just on any ol' block that it's registered for). Using "pattern" in the slug just muddies the waters a bit and will limit the use cases that folks imagine. Keep it to the more generic `section` prefix and be done with it.

Reply:
Alrighty, just asking to see what you think. Thanks!